defaults/helpers.go

```diff
@@ -18,3 +33,188 @@ func ociProfileConversionCheck(profile secprofile.Profile, entitlementID string)
 
 	return ociProfile, nil
 }
+
+// FIXME factor for all defaults package and even project-wide
+func testSpec() *specs.Spec {
+	s := &specs.Spec{
+		Process: specs.Process{
+			Capabilities: &specs.LinuxCapabilities{
+				Bounding:    []string{},
+				Effective:   []string{},
+				Inheritable: []string{},
+				Permitted:   []string{},
+				Ambient:     []string{},
+			},
+		},
+		Linux: &specs.Linux{
+			Seccomp:   &specs.LinuxSeccomp{},
+			Resources: &specs.LinuxResources{},
+			IntelRdt:  &specs.LinuxIntelRdt{},
+		},
+		Windows: &specs.Windows{
+			Resources: &specs.WindowsResources{
+				Memory:  &specs.WindowsMemoryResources{},
+				CPU:     &specs.WindowsCPUResources{},
+				Storage: &specs.WindowsStorageResources{},
+				Network: &specs.WindowsNetworkResources{},
+			},
+		},
+	}
+
+	seccomp, err := setDefaultSeccompProfile()
+	if err != nil {
+		// In case we get an error before seccomp struct is fully updated from decoded json, we empty it manually.
+		s.Linux.Seccomp = &specs.LinuxSeccomp{}
+	}
+
+	s.Linux.Seccomp = seccomp
+
+	return s
+}
+
+// We can't create a map to check set inclusion with specs.LinuxSeccompArg as key type in Golang so no O(n)
+func seccompArgsContains(syscallArgsSet, syscallArgsSubset []specs.LinuxSeccompArg) bool {
+	if len(syscallArgsSet) < len(syscallArgsSubset) ||
+		(len(syscallArgsSubset) == 0 && len(syscallArgsSet) != 0) {
```

Comment:
for `len(syscallArgsSubset) == 0 && len(syscallArgsSet) !=0` - isn't that a true case?  Aren't we trying to check if the `syscallArgsSet` contains all of the `syscallArgsSubset`?

Side note: might help to rename vars to better differentiate the master set vs. the subset to check for

Reply:
I need to rename this function or make a wrapper around it because it's currently doing more than just checking if `syscallArgsSubset` is contained in `syscallArgsSet`.

Answer to the 1st question:
Yes in theory.. but..
No because `syscallArgSubset` contains the syscall args of the to-be-checked syscall. If there are no arguments, that means that we want to completely whitelist the syscall completely so the whitelisting rule should be absolute (hence no args either).

Not sure I'm clear there.. 
function should actually be called `seccompArgsMatched(syscallRuleArgs, syscallArgsToCheck..`